lib/Distribution/Sle/AgamaDevel.pm

```diff
@@ -13,13 +13,15 @@ use warnings FATAL => 'all';
 use parent 'Distribution::Opensuse::Leap::16Latest';
 
 use Yam::Agama::Pom::GrubMenuBasePage;
+use Yam::Agama::Pom::GrubMenuBasePageBug1230121;
 use Yam::Agama::Pom::GrubMenuSlesPage;
 use Yam::Agama::Pom::AgamaUpAndRunningBasePage;
 use Yam::Agama::Pom::AgamaUpAndRunningSlePage;
 
 sub get_grub_menu_installed_system {
-    return Yam::Agama::Pom::GrubMenuSlesPage->new({
-            grub_menu_base => Yam::Agama::Pom::GrubMenuBasePage->new()});
+    return get_var('WORKAROUND_BSC_1230121') ?
```

Comment:
we should avoid to invent settings at all cost, they are really hard to maintain, in this case I guess you don't need a condition, it always fails now for sles.